lib/dpl/providers/elasticbeanstalk.rb

```diff
@@ -155,6 +172,10 @@ def bucket
       def eb
         @eb ||= Aws::ElasticBeanstalk::Client.new(retry_limit: 10)
       end
+
+      def debug(*args)
```

Comment:
Shouldn't this be more general (e.g., use logger with various log levels, and `debug` is just a synonym for `log_level=debug`)?

Reply:
that's a cool idea. would require some work going through other providers that use this same technique.